.github/workflows/ci.yml

```
@@ -34,6 +34,20 @@ jobs:
       - run: pip install black
       - run: black --check --diff modin/ asv_bench/benchmarks scripts/doc_checker.py
 
+  build-docs:
+    name: build docs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.7.x"
+          architecture: "x64"
+      - run: pip install -r docs/requirements-doc.txt
+      - run: cd docs && sphinx-build -b html . build
```

```suggestion
      - run: cd docs && sphinx-build -T -E -b html . build
```
1. `-T` should [increase verbosity](https://www.sphinx-doc.org/en/master/man/sphinx-build.html#cmdoption-sphinx-build-T) if something goes wrong.
2. `-E` [ensures](https://www.sphinx-doc.org/en/master/man/sphinx-build.html#cmdoption-sphinx-build-E) that we're not using any cache.